css/main.css

```diff
@@ -73,16 +79,18 @@ header{
 main{
     overflow: hidden;
     display: flex;
+    flex: 1;
 }
 
 .button-controls{
     width: 2em;
+    opacity: 0.6;
 }
 .button-controls img{
     width: 100%;
 }
 
-.button-round{
+.button_round{
```

Comment:
why underscore?

Reply:
using BEM convention, modifier should separated with _